### PR TITLE
hwatch 0.3.5 (new formula)

### PR DESCRIPTION
**Pull request by equal-l2**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
I couldn't come up with a good test case because `hwatch` is completely interactive, so any suggestion is welcome.

### CODE REVIEW

File: Formula/hwatch.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  def install
+    system "cargo", "install", *std_cargo_args
+    man1.install "man/hwatch.1"
+    bash_completion.install "completion/bash/hwatch-completion.bash"
+    zsh_completion.install "completion/zsh/_hwatch"
+    fish_completion.install "completion/fish/hwatch.fish"
+  end
+
+  test do
+    assert_match "hwatch #{version}", shell_output("#{bin}/hwatch --version")
```
Comment (chenrui333) [inline Formula/hwatch.rb:20]: Since this is a modern alternative to `watch`, it would be re-use the test from `watch` formula.

Comment (equal-l2) [reply Formula/hwatch.rb:20]: Unfortunately, we can't.

`watch` has options such as `--errexit` (exit on command error) or `--chgexit` (exit on change), but `hwatch` doesn't.

Comment (equal-l2) [reply Formula/hwatch.rb:20]: Maybe I should suggest implementing those features to the author.

Comment (chenrui333) [reply Formula/hwatch.rb:28]: For now, I just use a spawned thread to run the hwatch, it seems working fine.

Comment (chenrui333) [reply Formula/hwatch.rb:28]: yeah, can you create some feature request for `hwatch` author to make it on par with `watch`? Thanks!

File: Formula/hwatch.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  sha256 "e01bf716db8d1561f79d12c178df3b48e927957c6c595b841ba2b1bc4faa760e"
+  license "MIT"
+  head "https://github.com/blacknon/hwatch.git", branch: "master"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+    man1.install "man/hwatch.1"
+    bash_completion.install "completion/bash/hwatch-completion.bash"
```
Comment (chenrui333) [inline Formula/hwatch.rb:14]:
```suggestion
    bash_completion.install "completion/bash/hwatch-completion.bash" => "hwatch"
```

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@equal-l2, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**